src/lib/resource/interceptor/refresh.es

```diff
@@ -101,25 +104,34 @@ class TokenRefresher {
    * @param {number} opts.expires_in
   **/
   updateToken (opts = {}) {
-    if (!this.session) return
-    this.session.token(opts.access_token)
-    this.session.expiresIn(opts.expires_in)
-    this.session.refreshToken(opts.refresh_token)
+    if (!opts.token) {
+      this._clear()
+      throw new Error('TokenRefresher: Error retrieving the new token')
+    }
+
+    let gSession = window.syn.auth.session.global.get()
```

Comment:
I have to save it into the localStorage in order to not load the old token next time the page is reloaded, does this response to your question?